Dockerfile

```diff
@@ -1,19 +1,17 @@
 # This file is part of REANA.
-# Copyright (C) 2018, 2019, 2020, 2021, 2022, 2023 CERN.
+# Copyright (C) 2018, 2019, 2020, 2021, 2022, 2023, 2024 CERN.
 #
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
 # Use Ubuntu LTS base image
-FROM docker.io/library/ubuntu:20.04
+FROM docker.io/library/ubuntu:24.04
```

Comment:
BTW please try to update the other occurrences of Ubuntu base version so that we standardise everywhere on Ubuntu 24.04:

```console
$ rg ubuntu
• Dockerfile
8:FROM docker.io/library/ubuntu:24.04

• .readthedocs.yaml
10:  os: ubuntu-22.04

• .github/workflows/ci.yml
13:    runs-on: ubuntu-20.04
39:    runs-on: ubuntu-20.04
50:    runs-on: ubuntu-20.04
67:    runs-on: ubuntu-20.04
84:    runs-on: ubuntu-20.04
101:    runs-on: ubuntu-20.04
118:    runs-on: ubuntu-20.04
143:    runs-on: ubuntu-20.04
169:    runs-on: ubuntu-20.04
178:    runs-on: ubuntu-20.04
187:    runs-on: ubuntu-20.04

• .github/workflows/release-please.yml
15:    runs-on: ubuntu-latest
```